crates/cmds-solana/src/wormhole/token_bridge/eth/create_wrapped_on_eth.rs

```diff
@@ -158,7 +158,6 @@ mod tests {
             write!(&mut s, "{:02x}", b).unwrap();
         }
         dbg!(s);
-
-        panic!("")
+        Ok(())
```

Comment:
@enzotar are you trying to see the logs?

you can pass `--nocapture` like this to see logs of tests 
```
cargo test -- --nocapture
```